### DISCUSSION TITLE
when the application starts, it issues an error about microsoft.net

### DISCUSSION THREAD

**Discussion by armageddon735** [killkimno/MORT#58] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/37671417-2ea5-4f26-b6d2-1fde5f2dd829)
I have already tried to install the specified file from the official website, but the application still does not start, giving the same error.

**Comment by kmonkeyhead** [comment]
This error occurs when the .net7 runtime package is missing.
Please check if it is installed properly.

and try rebooting.

**Reply by armageddon735** [reply]
I don't know much about such things, but there were no errors during the installation, is there a way to check if everything is in order?

**Reply by armageddon735** [reply]
or maybe I just downloaded the wrong thing. I downloaded this one specifically.:
![image](https://github.com/user-attachments/assets/34734b33-00e3-48a3-90f4-3d4ef4553b1b)

**Reply by kmonkeyhead** [reply, marked as answer]
First of all, is the OS you are using 64-bit?
If not, please use MORT 32BIT version
https://blog.naver.com/killkimno/222936631523

And try install this package
https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/runtime-desktop-7.0.8-windows-x64-installer

**Reply by armageddon735** [reply]
that's right, my OS is 64-bit.
And thanks for the link, I'll try it now.

**Reply by armageddon735** [reply]
Everything worked.
It looks like I downloaded the wrong ".net" after all.
Thank you so much for your help!
